### DISCUSSION TITLE
Frient ZHEMI101 reporting interval

### DISCUSSION THREAD

**Discussion by KHV8** [Koenkk/zigbee2mqtt#12203] [Devices]
Have been using the energy meeter since september, working as intented. I have recently moved to Z2M from ZHA in Home Assistant. The energy meeter is using up a big part of the HA Database, currently close to ½ the space for "states". this was similar when using ZHA, so a general problem with the ZHEMI101 due to numerous reading, like every 5 seconds.

When I was using ZHA I created a template sensor which every 5 minutes made a copy of the energy value, giving me the value used for the energy dashboard. I then disabled storing of entities from ZHEMI101 in recorder. I can do the same with the Z2M setup, however maybe a simple way.

I have tried to change the "reporting" values for the device. They are default to 5 seconds as minimum value and 1 second for "change". I have tried to change them to different values, like the one below.
![image](https://user-images.githubusercontent.com/82907824/163731555-87d2c61f-d166-4295-b293-ffc48598fede.png)

However, it does not make any changes. The ZHEMI101 reports every 5 or 10 or 15 seconds. Never any other interval.

Have I misunderstood the possibility to change the reporting periods (meaning that it is not possible at all to change) or is there something else wrong 8something I do not understand/do right)?

Forgot some basis information: 
HA, latest version, Z2M version 1.25, Sonoff Zigbee 3.0 dongle with 20220219 FW. Total zigbee setup with 70+ devices, working with no problems.

**Comment by DML1972** [comment]
I don't have a solution but I am stuck on how to get the device paired. Any help? Thanks

**Reply by sjorge** [reply]
If you want updates exactly ever 5 minutes you probably want
min=300 max=300 change=0

By setting both the min and max to 5 minutes and change to 0 it will always do an update after 5 minutes and not before.

Although it's also possible the device ignores the reporting configuration.
